library/public/types.h

```diff
@@ -1,14 +1,16 @@
 #ifndef f3d_types_h
 #define f3d_types_h
 
+#include "export.h"
```

Comment:
revert changes in this file

Reply:
it is needed, fails on windows without it

Reply:
https://github.com/f3d-app/f3d/actions/runs/5141210018/jobs/9253451184